markdown_it/rules_block/blockquote.py

```diff
@@ -296,6 +296,4 @@ def blockquote(state: StateBlock, startLine: int, endLine: int, silent: bool):
 
     state.blkIndent = oldIndent
 
-    state.lineMax += 1
```

Comment:
This increment doesn't exist in JS upstream so removing it https://github.com/markdown-it/markdown-it/blob/750a954d586c526e92bc4305bde2552743304d13/lib/rules_block/blockquote.js#L282